index.js

```diff
@@ -74,6 +53,16 @@ class BabelCompiler {
     this.options.filename = file.path;
     this.options.sourceFileName = file.path;
 
+    if (!this.options.presets) {
+      const babelConfig = new babel.OptionManager().init(this.options);
+      console.log(babelConfig);
+      const hasConfig = babelConfig.presets && babelConfig.presets.length > 0;
```

Comment:
Had seen Logan's comment later. Agree.
If you want to provide `env` as a default preset, how do you want to people configure it? The real charm of `env` preset is to use it with targets. Maybe with brunch-config?